### PR TITLE
ZEN-28136 UI states that Maintenance times will be in UTC

### PR DESCRIPTION
**Pull request by ISPHOST**

### CODE REVIEW

**Comment by jpeacock-zenoss** [comment]
It looks like you've changed existing APIs to return a value of a type that's different from what those APIs returned in prior releases.  In general, this is not a good idea.

File: Products/ZenModel/MaintenanceWindow.py [hunk 1]
```diff
@@ -239,21 +240,24 @@ def makeInt(v, fieldName, minv=None, maxv=None, acceptBlanks=True):
...
-            month, day, year = re.split('[^ 0-9]', startDate)
-        except ValueError:
-            msgs.append("Date needs three number fields")
-        day = int(day)
-        month = int(month)
-        year = int(year)
-        if not msgs:
-            t = time.mktime((year, month, day, startHours, startMinutes,
+        if startDateTime:
+            t = startDateTime
```
Comment (jpeacock-zenoss) [inline Products/ZenModel/MaintenanceWindow.py]: There should be code here to make sure that `t` references an integer type.  Something like:
```
    t = int(startDateTime)
```
This code will work whether `startDateTime` is an `int` or a string representing an integer.

File: Products/ZenModel/tests/testMaintenanceWindow.py [hunk 2]
```diff
@@ -158,9 +158,7 @@ def testMaintenanceWindows(self):
         #r = FakeRequest()
         r = None
         m.manage_editMaintenanceWindow(
-                                         startDate='01/29/2006',
-                                         startHours='10',
-                                         startMinutes='45',
+                                         startDateTime='1138531500',
```
Comment (jpeacock-zenoss) [inline Products/ZenModel/tests/testMaintenanceWindow.py]: To ensure the test is consistent, the `startDateTime` value should be 1138553100.
That's the value I get when I plug the original values into `time.mktime`:
```
>>> time.mktime((2006, 1, 29, 10, 45, 0, 0, 0, -1))
1138553100.0
>>>
```

File: Products/ZenModel/tests/testMaintenanceWindow.py [hunk 3]
```diff
@@ -171,7 +169,7 @@ def testMaintenanceWindows(self):
 
         #self.assert_('message' in r)
         #self.assert_(r['message'] == 'Saved Changes')
-        self.assert_(m.start == t - 12)
+        #self.assert_(m.start == t - 12)
```
Comment (jpeacock-zenoss) [inline Products/ZenModel/tests/testMaintenanceWindow.py]: Please do not comment out this line.  This is verifying that `manage_editMaintenanceWindow` has updated the `start` attribute correctly.

File: Products/ZenModel/tests/testMaintenanceWindow.py (hunk 2 above)
Comment (ISPHOST) [reply Products/ZenModel/tests/testMaintenanceWindow.py]: If you run this code in zope container you will get
>>> time.mktime((2006, 1, 29, 10, 45, 0, 0, 0, -1))
1138531500.0

**Pull request merged**